lib/OpenQA/Worker/Isotovideo/Client.pm

```diff
@@ -1,11 +1,11 @@
-# Copyright 2019 SUSE LLC
+# Copyright 2019-2021 SUSE LLC
```

Comment:
please leave the old. Our approach should be to not update Copyright anymore at all after the initial year